src/Components/Nav/NavSort.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import DotCMSApi from '../../libs/dotcms.api';
+import PageContext from '../../PageContext';
+
+class NavSort extends Component {
+    static contextType = PageContext;
+
+    emitEvent() {
+        const siteId = this.context.site ? this.context.site.identifier : '';
+        const pageUrl = this.context.page ? this.context.page.pageURI : '';
+        const PORLET_ID = 'b7ab5d3c-5ee0-4195-a17e-8f5579d718dd';
+        const url = [
+            `/c/portal/layout?p_l_id=${PORLET_ID}&p_p_id=site-browser`,
+            `&p_p_action=1&p_p_state=maximized&_site_browser_struts_action=%2Fext%2Ffolders%2Forder_menu`,
+            `&startLevel=1&depth=2&pagePath=${pageUrl}&hostId=${siteId}`
+        ].join('');
+
+        DotCMSApi.page.emitCustomEvent('reorder-menu', url, 'ng-event');
+    }
+
+    render() {
+        return (
+            <button className="nav-sort__button btn btn-light" onClick={() => this.emitEvent()}>
```

Comment:
don't need arrow func.

Reply:
it is needed, otherwise this https://github.com/dotCMS/dotcms-spa-edit-mode/blob/81d2d23096a877697cece13ae532e7a5ca2721ce/src/Components/Nav/NavSort.js#L9
throws `undefined`